streams/us.m3u

```diff
@@ -311,8 +301,6 @@ https://cbsta49f-cbsta49f-ms.global.ssl.fastly.net/amagi7b98-AmagiMixible/master
 https://eyeonesports.com/ES2RA-628g.m3u8
 #EXTINF:-1 tvg-id="EternalLifeTVNetwork.us",Eternal Life TV Network (1080p)
 https://livecdn.live247stream.com/eternallife/tv/playlist.m3u8
-#EXTINF:-1 tvg-id="",Everyday Refresh
-https://qvc-amd-live.akamaized.net/hls/live/2034113/lsqvc6us/master.m3u8
```

Comment:
working link
<img width="1404" alt="image" src="https://github.com/user-attachments/assets/ceff21bf-c2ab-4365-b60d-716f0039d2ef">

Reply:
QVC Has news channels for the holidays I guess and some links have change.
Holiday Headquarters (Everyday Refresh)
https://qvc-amd-live.akamaized.net/hls/live/2034113/lsqvc6us/master.m3u8
The Retail Therapy Channel
https://qvc-amd-live.akamaized.net/hls/live/2034113/lsqvc8us/master.m3u8